.circleci/config.yml

```
@@ -30,52 +30,59 @@ jobs:
             # NOTE: blas multithreading behaves badly on circleci
             export OMP_NUM_THREADS=1
             source venv/bin/activate
-            # n = nitpicky (broken links), W = warnings as errors,
-            # T = full tracebacks, keep-going = run to completion even with errors
-            make -C site/ SPHINXOPTS="-nWT --keep-going" html
+            myst build --html --execute
```

Maybe something like this will work:

```suggestion
            # Construct the BASE_URL using the CIRCLE_WORKFLOW_JOB_ID
            export BASE_URL="/output/job/${CIRCLE_WORKFLOW_JOB_ID}/artifacts/0/_build/html"

            # Output the BASE_URL for debugging purposes
            echo "BASE_URL is set to $BASE_URL"

            # Run your myst build command
            myst build --html --execute
```



See https://github.com/executablebooks/mystmd/issues/1269, https://github.com/executablebooks/mystmd/issues/1270